client/public/index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <script src="https://www.paypal.com/sdk/js?client-id=AY7O6M0NDbBh3f6eaRpynKmm5v7KUgf6pWaKXJIr3UY0i10x5uPB9a6CmjUWlWD-jpZ8HWXJFuJq03fL"></script>
```

Comment:
do you still need this script?

Reply:
Whenever I tried removing the script it would make the page not load, it did not crash however.